crates/nu-command/src/platform/ansi/ansi_.rs

```diff
@@ -16,212 +16,802 @@ struct AnsiCode {
     code: String,
 }
 
-lazy_static! {
-    static ref CODE_LIST: Vec<AnsiCode> = vec!{
-    AnsiCode{ short_name: Some("g"), long_name: "green", code: Color::Green.prefix().to_string()},
-    AnsiCode{ short_name: Some("gb"), long_name: "green_bold", code: Color::Green.bold().prefix().to_string()},
-    AnsiCode{ short_name: Some("gu"), long_name: "green_underline", code: Color::Green.underline().prefix().to_string()},
-    AnsiCode{ short_name: Some("gi"), long_name: "green_italic", code: Color::Green.italic().prefix().to_string()},
-    AnsiCode{ short_name: Some("gd"), long_name: "green_dimmed", code: Color::Green.dimmed().prefix().to_string()},
-    AnsiCode{ short_name: Some("gr"), long_name: "green_reverse", code: Color::Green.reverse().prefix().to_string()},
-    AnsiCode{ short_name: Some("bg_g"), long_name: "bg_green", code: Style::new().on(Color::Green).prefix().to_string()},
+static CODE_LIST: Lazy<Vec<AnsiCode>> = Lazy::new(|| {
+    vec![
+        AnsiCode {
+            short_name: Some("g"),
+            long_name: "green",
+            code: Color::Green.prefix().to_string(),
+        },
+        AnsiCode {
+            short_name: Some("gb"),
+            long_name: "green_bold",
+            code: Color::Green.bold().prefix().to_string(),
+        },
```

Comment:
It's difficult to review these changes, and the code is harder to read now that every `AnsiCode` takes up 5 lines. Can you change it back to 1 line per `AnsiCode`? You may be able to use `#[rustfmt::skip]`

Reply:
fixed it, thanks for the feedback